js/cryptoLib/index.js

```diff
@@ -142,36 +132,37 @@ class Crypto {
                         reject(new Error(err));
                     }
                     fs.unlink(`${this.dump}/decrypted.zip`, () => {
-                        fs.unlink(this.encryptedIndex, () => {
-                            resolve('success');
-                        })
+                        resolve('success');
                     });
                 })
             }
         });
     }
 
     /**
+     * Deleting the data index folder
+     * when the app is closed
+     */
+    deleteFolders() {
+        Crypto.deleteFolderRecursive(this.dataFolder);
```

Comment:
Why do we need to delete folders recursively?

My assumption is -> While closing the app, we would only have one user's directory in the data folder, correct?

Reply:
We cannot delete a directory that is not empty, So to remove a dir we need to recursively remove all the files and folder inside it.